azkaban-exec-server/src/main/java/azkaban/common/ServerUtils.java

```diff
@@ -65,4 +68,16 @@ public static String getVersionSetJsonString(final VersionSet versionSet) {
     }
     return JSONUtils.toJSON(imageToVersionStringMap, true).replaceAll("\"", "");
   }
+
+  public static void setupTimeZone(final Props azkabanSettings, @Nonnull final Logger logger) {
+    if (azkabanSettings.containsKey(Constants.ConfigurationKeys.DEFAULT_TIMEZONE_ID)) {
```

Comment:
This is also done in `AzkabanWebServer` class. Could you move this method to `AzkabanServer` class and call it from `AzkabaWebServer`, `AzkabanExecutorServer` and `FlowContainer`)?

Reply:
Good point. working on it.